.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
you are supposed to revert everything i.e., auto generated by android studio 
simply everything under .idea folder
thanks@Prince-kushwaha